packages/react-search-ui-views/src/Autocomplete.js

```diff
@@ -1,34 +1,8 @@
 import PropTypes from "prop-types";
 import React from "react";
 
-import { Result } from "./types";
-import { Suggestion } from "./types";
-import { appendClassName } from "./view-helpers";
-
-function getRaw(result, value) {
-  if (!result[value] || !result[value].raw) return;
-  return result[value].raw;
-}
-
-function getSnippet(result, value) {
-  if (!result[value] || !result[value].snippet) return;
-  return result[value].snippet;
-}
-
-function getSuggestionTitle(suggestionType, autocompleteSuggestions) {
-  if (autocompleteSuggestions.sectionTitle) {
-    return autocompleteSuggestions.sectionTitle;
-  }
-
-  if (
-    autocompleteSuggestions[suggestionType] &&
-    autocompleteSuggestions[suggestionType].sectionTitle
-  ) {
-    return autocompleteSuggestions[suggestionType].sectionTitle;
-  }
-}
-
 function Autocomplete({
+  viewHelpers,
```

Comment:
Just to check my high level understanding of this PR: why move this to a passed down prop instead of either 1. importing it or 2. using a hook?

I sort of get it if the goal is to expose `viewHelpers` to end-developers, but I also still hold that a prop isn't the most elegant way to do that - I think devs would prefer it to be a helper/hook they can choose to import only if they want it.

Reply:
After rereading the examples in the PR description and mulling on it more, I see the argument for a passed prop - I'm still leaning a little towards thinking a hook is a bit nicer/more modern, but don't want to block this PR/release for that. 👍 from me if others are on board

Reply:
Yeah, just to re-iterate, many people create custom views by copying the existing views from source, so like https://github.com/elastic/search-ui/blob/master/packages/react-search-ui-views/src/Result.js.

The following trips them up:
```
import { appendClassName, getUrlSanitizer } from "./view-helpers";
```

That's a relative import from within the project. When they use it externally, it's confusing if / how they can import those view helpers themselves.

Passing them as a prop is just a very low friction way of addressing this. They don't have to figure out how to import anything.

Your point about hooks is totally valid, though I think that would still require an import. We currently can't use hooks in views though 🤦 : https://github.com/elastic/search-ui/issues/276.

There may be a way though, through some build magic that we could change the import to something like:

```
import { appendClassName, getUrlSanitizer } from "@elastic/react-search-ui-views/view-helpers";
```

In such a way that it works internal to the project and external.

I'm not married to this approach so if there is clearly a more elegant approach I am open to it.

Reply:
I'd definitely be +1 to a non-relative import like `import { appendClassName, getUrlSanitizer } from "@elastic/react-search-ui-views/view-helpers";` - that matches a lot of what other React libraries do as a best practice.

I guess the question is, would you rather merge this PR is as-is and change the prop/import behavior later in a major/breaking release, or would you be rather either delay the release or punt the PR to another release?